source/PlayerInfoPanel.cpp

```diff
@@ -924,3 +956,10 @@ PlayerInfoPanel::SortableColumn::SortableColumn(
 	: name(name), offset(offset), endX(endX), layout(layout), shipSort(shipSort)
 {
 }
+
+
+
+string PlayerInfoPanel::SortableColumn::Tooltip() const
```

Comment:
This should be returning a const reference.
```suggestion
const string &PlayerInfoPanel::SortableColumn::Tooltip() const
```